src/mmw/js/src/modeling/controllers.js

```diff
@@ -35,26 +38,29 @@ var ModelingController = {
                 name: 'Untitled Project',
                 created_at: Date.now(),
                 area_of_interest: App.map.get('areaOfInterest'),
-                scenarios: new models.ScenariosCollection([
-                    new models.ScenarioModel({
-                        name: 'Current Conditions',
-                        is_current_conditions: true,
-                        active: true
-                    })
-                ])
+                scenarios: new models.ScenariosCollection()
+            });
+
+            App.currProject = project;
+
+            var currentConditionsScenario = new models.ScenarioModel({
+                name: 'Current Conditions',
+                is_current_conditions: true,
+                active: true
             });
+            project.get('scenarios').add(currentConditionsScenario);
+
+            initViews(project);
 
             project.on('change:id', function(model) {
                 router.navigate('model/' + model.id);
             });
 
-            var scenariosColl = project.get('scenarios'),
+            var scenarios = project.get('scenarios'),
                 mapView = App.getMapView();
-            scenariosColl.on('change:activeScenario', function(scenario) {
+            scenarios.on('change:activeScenario', function(scenario) {
```

Comment:
I prefer singular vs plural to distinguish models and collections because it's more succinct and matches natural language well. We're also using it in `modeling/models.js` in the attribute names `results` and `scenarios` and also in view names in `modeling/views.py`.